core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -534,16 +535,16 @@ object Chunk {
         override def map[O2](f: O => O2): Chunk[O2] = indexedSeq(s.map(f))
       }
 
-  /** Creates a chunk backed by a `Seq`. */
-  def seq[O](s: Seq[O]): Chunk[O] = s match {
+  /** Creates a chunk backed by a mutable `Seq`. */
```

Comment:
A collection.Seq an be either mutable _or_ immutable, right?

Reply:
Correct. It is a parent of both `collection.immutable.Seq` and `collection.mutable.Seq`.

Reply:
Fixed